src/flamegraph/merge.rs

```diff
@@ -148,8 +172,29 @@ where
             iter::once("").chain(last.split(';')),
             None,
             time,
+            delta,
         );
     }
 
-    (frames, time, ignored)
+    (frames, time, ignored, delta_max)
+}
+
+// Parse and remove the number of samples from the end of a line.
+fn parse_nsamples(line: &mut &str) -> Option<usize> {
+    if let Some(samplesi) = line.rfind(' ') {
+        let mut samples = &line[(samplesi + 1)..];
+        // strip fractional part (if any);
+        // foobar 1.klwdjlakdj
```

Comment:
While we're touching this code, would you mind adding a reference to https://github.com/jonhoo/inferno/issues/43 here?